Testing/Data/DocTest/EMU00103638.nxs_v2.md5

```diff
@@ -1 +0,0 @@
-7a1a43edf3e0172536c9995449e41ba5
```

Comment:
Can I delete the binary from the storage on the server?

Reply:
Yes please do, thank you.

Reply:
Thanks. Done.